src/components/endringslogg/utils/endringslogg-custom.tsx

```diff
@@ -21,6 +21,26 @@ export interface EndringsloggInnleggMedSettStatus extends EndringsloggInnlegg {
 }
 
 const endringslogginnhold: EndringsloggInnlegg[] = [
+    {
+        dato: '16. november 2020',
+        tittel: 'Filtre på svar fra registrering om utdanning',
+        featureToggleName: 'veilarbportefolje.utdanning_filter',
+        versjonId: '16.11.20',
+        children: (
+            <>
+                <p>
+                    Nå kan du filtrere på svarene brukere oppga i registreringen på spørsmålene om utdanning. Dette kan
```

Comment:
Her er full tekst:

Nå kan du filtrere på svarene brukerne oppga i registreringen på spørsmålene om utdanning. Dette kan hjelpe deg med å identifisere og prioritere hvilke brukere som kan ha behov for veiledning om utdanning.

Filtrene ligger i fanen lengst til høyre under "Svar fra registrering".